subscriptions/client/database/database_test.go

```diff
@@ -0,0 +1,34 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"main/config"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConnString(t *testing.T) {
+	t.Run("config", func(t *testing.T) {
+		config, err := config.LoadConfig()
```

Comment:
i guess it would not work unless you add env variables before starting the test. probably for the purpose of testing just ConnString function you could define config entity here and remove LoadConfig call. Probably LoadConfig should be tested separately